libvmaf/src/output.h

```diff
@@ -19,6 +19,8 @@
 #ifndef __VMAF_OUTPUT_H__
 #define __VMAF_OUTPUT_H__
 
+int countLeadingZeroesFloat(float x);
```

Comment:
I think this function could be `static` and then the header doesn't need to declare it